exercises/07-normalizr/src/modules/api/api-client.ts

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios'
+
+export const apiClient = axios.create({
+  baseURL: 'http://localhost:3000',
```

Comment:
i'd move the whole rest api underneath /api path.

Reply:
I'm fine with it like it is - as far as I know, the backend only serves as an API and in real project there would be some nginx proxy (ehm gateway), that would handle that. Here in training it'd only complicate things more. Thoughts?